client/ayon_core/tools/publisher/widgets/create_widget.py

```diff
@@ -328,9 +328,6 @@ def _get_task_name(self):
             folder_path = self._context_widget.get_selected_folder_path()
             if folder_path:
                 task_name = self._context_widget.get_selected_task_name()
-
```

Comment:
BTW `_get_folder_path` also returns current folder path if nothing is selected. I wonder if there was a reason to do this?

Reply:
Yes, but in that case `None` is passed to them (AFAIK). So it doesn't seem to have a reason...

Reply:
Fixed - seems to work as expected.